docs/examples/django/README.rst

```diff
@@ -105,6 +106,14 @@ Django's instrumentation can be disabled by setting the following environment va
 
 #. ``export OTEL_PYTHON_DJANGO_INSTRUMENT=False``
 
+Auto Instrumentation
+--------------------
+
+This same example can be run using auto instrumentation. Comment out the call
+to ``DjangoInstrumento().instrument()`` in ``main``, then Run the django app
+with ``opentelemetry-instrumentation python manage.py runserver --noreload``.
+Repeat the steps with the client, the result should be the same.
```

Comment:
Generally people don't use `runserver` in real environment and only use it for dev. I wonder if we should document for local/dev environment like this. If we want it, I'd suggest to change title to `Auto Instrumentation for runserver` to clarify production envs shouldn't do this.

Reply:
Read the rest of the document. This whole doc should be re-structured to contain info on how to instrument in at least three well-known cases which are "runserver" (for local dev), "gunicorn" and "uwsgi".

This PR doesn't need to do that of course but would be nice if we had an issue for it.

Reply:
Added clarification :+1: